lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1497,6 +1505,70 @@ def test_teamset_type(self, requesting_user, expected_status):
             self.assertEqual(team['name'], 'foo')
 
 
+@ddt.ddt
+class TestTeamAssignmentsView(TeamAPITestCase):
+    """ Tests for the TeamAssignmentsView """
+
+    @classmethod
+    def setUpClass(cls):
+        """ Create an openassessment block for testing """
+        super(TestTeamAssignmentsView, cls).setUpClass()
```

Comment:
Nice use of setupClass! Btw, there is no need to use class name in super calls in python 3. It can be:
`super().setUpClass()`

Reply:
Super, thanks!

Note: Pun intended